src/pe/debug.rs

```diff
@@ -0,0 +1,120 @@
+use scroll::{self, Pread};
+use error;
+
+use pe::section_table;
+use pe::utils;
+use pe::data_directories;
+
+#[derive(Debug, PartialEq, Copy, Clone, Default)]
+pub struct DebugData<'a> {
+    pub image_debug_directory: ImageDebugDirectory,
+    pub codeview_pdb70_debug_info: Option<CodeviewPDB70DebugInfo<'a>>,
+}
+
+impl<'a> DebugData<'a> {
+    pub fn parse(bytes: &'a [u8], dd: &data_directories::DataDirectory, sections: &[section_table::SectionTable]) -> error::Result<Self> {
+        let image_debug_directory = ImageDebugDirectory::parse(bytes, dd, sections)?;
+        let codeview_pdb70_debug_info = CodeviewPDB70DebugInfo::parse(bytes, &image_debug_directory)?;
+
+        Ok(DebugData{
+            image_debug_directory,
```

Comment:
I was trying to keep 1.15 compat; I can't remember when shorthand initializers were introduced. If it was like 1.16 or 1.17 I'm maybe ok with bumping since that's somewhat back and the ergonomics are worth it but I did like the 1.15 commitment...